.github/workflows/lint-pr-title.yml

```diff
@@ -16,6 +16,6 @@ jobs:
   lint-pr-title:
     runs-on: ubuntu-24.04
     steps:
-      - uses: grafana/shared-workflows/actions/lint-pr-title@90e72fd7b35f5d30696313aeb736a13a15eb82ad # lint-pr-title-v1.0.0
+      - uses: grafana/shared-workflows/actions/lint-pr-title@13fb504e3bfe323c1188bf244970d94b2d336e86 # lint-pr-title-v1.0.0
```

Comment:
This update looks weird as it points to the commit that released get-vault-secrets-v1.0.1. Does dependabot perhaps not support the comment style here?

Reply:
Should probably be OK to use the default semver comments since we can get the name of the action from the path?

Reply:
It's because the actions can all be released separately. We're trying to avoid having updates for consumers when some other action changed and not the one they're using... but it doesn't work this way 😢